src/prefect/cli/kv_store.py

```diff
@@ -0,0 +1,96 @@
+import sys
```

Comment:
I know there's a lot of boilerplate in these commands for logging and exiting. I'd like to refactor them when we create proper abstractions out of the `build_register` decorator / `TerminalError` work.